src/HexBug/utils/mods.py

```diff
@@ -330,6 +339,32 @@ class RegistryMod(Enum):
         modloaders=[FABRIC, QUILT],
     )
 
+    Mediaworks = RegistryModInfo(
+        name="Mediaworks",
+        description="Adds QOL features and new spells. Create HUD's and become a ghost!",  # stolen from addons.hexxy.media
+        directory="vendor/mediaworks",
+        book=mediaworks_docgen.parse_book(
+            "vendor/mediaworks/common/src/main/resources",
+            "vendor/mediaworks/doc/resources",
+            "mediaworks",
+            "mediaworksbook",
+        ),
+        registry_regex_type=RegistryRegexType.Mediaworks,
+        book_url="https://artynova.github.io/mediaworks/",
+        curseforge_url=None,
+        modrinth_slug="mediaworks",
+        source_url="https://github.com/artynova/mediaworks",
```

Comment:
`/source` expects a trailing slash here.
```suggestion
        source_url="https://github.com/artynova/mediaworks/",
```